src/main/resources/assets/tfc/lang/en_us.lang

```diff
@@ -3826,4 +3826,137 @@ tfc.animal.horsetfc.female=Horse♀
 tfc.animal.donkeytfc.male=Donkey♂
 tfc.animal.donkeytfc.female=Donkey♀
 tfc.animal.muletfc.male=Mule♂
-tfc.animal.muletfc.female=Mule♀
\ No newline at end of file
+tfc.animal.muletfc.female=Mule♀
+
+## Advancements
+
+# Main Quest
+# Stone Age
+advancements.main.root.title=TerraFirmaCraft
```

Comment:
To avoid lang files collision (where another mod / resource pack use the same lang key) everything within TFC's realm should have use `<prefix.if.needed>.tfc.<item.name>` standard. So, for advancements, i recommend:
`advancements.tfc.<tree>.<advancement>.title`